lnwallet/btcwallet/psbt.go

```diff
@@ -207,6 +222,61 @@ func validateSigningMethod(in *psbt.PInput) (input.SignMethod, error) {
 
 		return input.SegWitV0SignMethod, nil
 
+	case txscript.WitnessV1TaprootTy:
+		if len(in.TaprootBip32Derivation) == 0 {
+			return 0, fmt.Errorf("cannot sign for taproot input " +
+				"without taproot BIP0032 derivation info")
+		}
+
+		// Currently, we only support creating one signature per input.
+		//
+		// TODO(guggero): Should we support signing multiple paths at
```

Comment:
As in generating multiple signatures for a checksig_add case that uses a repeated key?